runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollOverlay.java

```diff
@@ -85,6 +94,15 @@ public Dimension render(Graphics2D graphics)
 			}
 		}
 
+		if (clue.isRequiresLight() && plugin.getInventoryItems() != null)
```

Comment:
There's a check for `getInventoryItems` being null, but not `getEquippedItems`

Reply:
Actually, the cases where the inventory is empty and equipped items are empty should be handled separately right?